editor/plugins/item_list_editor_plugin.h

```diff
@@ -158,6 +166,9 @@ class ItemListPopupMenuPlugin : public ItemListPlugin {
 	virtual void add_item() override;
 	virtual int get_item_count() const override;
 	virtual void erase(int p_idx) override;
+	virtual void undoRedo_add_new_item(int idx) override;
+	virtual void undoRedo_re_add_item(unsigned int position, String name, Ref<Texture2D> icon, bool enabled) override;
```

Comment:
PopupMenu's have a lot more information than just text, icon and enabled. There is also:

* Checkable: yes/no/as checkbox
* Checked: true/false
* Id
* Enabled: true/false
* Separator: true/false

This method in it's current state would lose a lot of that information.